packages/formation-react/src/components/AlertBox/AlertBox.jsx

```diff
@@ -38,6 +38,7 @@ class AlertBox extends React.Component {
     const alertClass = classNames(
       'usa-alert',
       `usa-alert-${this.props.status}`,
+      { 'background-color-only': this.props.backgroundOnly },
```

Comment:
I haven't seen this `background-color-only` class before - should it have the VADS prefix?

Reply:
I think it predates the `vads-` convention.